main.go

```diff
@@ -186,8 +188,11 @@ func dateParamToEpoch(s string, d int64) int32 {
 
 	for _, format := range timeFormats {
 		split := strings.Fields(s)
-		ts, ds := split[0], split[1]
+		if len(split) > 2 {
```

Comment:
Actually, this makes it clear that if `len(split)==1`, we'll panic down below.

Add `{"19940816", "00:00 1994-Aug-16"},` to your list of test cases.